MentalHospital.Tests/InitialMethods/PatientInitial.cs

```diff
@@ -0,0 +1,32 @@
+﻿using MentalHospital.BLL.Models;
+using MentalHospital.DAL.Entities;
+
+namespace MentalHospital.Tests.InitialMethods
```

Comment:
Rename folder name to the TestData or Data. Separate Entities and Models per folders